processing/src/main/java/org/apache/carbondata/processing/store/writer/v1/CarbonFactDataWriterImplV1.java

```diff
@@ -373,7 +373,7 @@ protected void writeBlockletInfoToFile(FileChannel channel, String filePath)
       FileFooter convertFileMeta = CarbonMetadataUtil
           .convertFileFooter(blockletInfoList, localCardinality.length, localCardinality,
               thriftColumnSchemaList, dataWriterVo.getSegmentProperties());
-      fillBlockIndexInfoDetails(convertFileMeta.getNum_rows(), filePath, currentPosition);
+      fillBlockIndexInfoDetails(convertFileMeta.getNum_rows(), carbonDataFileName, currentPosition);
```

Comment:
Please align the parameter name(filePath) for fillBlockIndexInfoDetails of AbstractFactDataWriter.java
For example : 
 protected void fillBlockIndexInfoDetails(long numberOfRows,
    String carbonDataFileName, long currentPosition)

Please modify accordingly for all part.